src/main/webapp/index.jsp

```diff
@@ -36,7 +36,7 @@
     <div
       style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
 
-      <h1>CodeU Chat App</h1>
+      <h1>CodeUUU Chat App</h1>
```

Comment:
Let's make this something we'd actually want to deploy. Perhaps, 'Team 34 Chat App' ?